content/posts/2021-features.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "2021 for Packit"
+date: 2022-01-12T09:23:38+01:00
+---
+
+# Packit project in 2021
+
+The previous year 2021 wasn't interesting only because of the increased usage of Packit
+(you can see more in [the previous post](../2021-in-numbers)).
+The whole Packit team made a lot of improvements during the year.
+Some small, some really big. So, let's take a look at the most important ones:
+
+## Dashboard
+
+The idea of having a dashboard for Packit service started as a Google Summer of Code 2020 project
+to provide a basic view of our service.
+Nowadays, it's a core part of the project and it has replaced the result pages in plain HTML. Do you remember them?
+
+The dashboard can be found at [dashboard.packit.dev](https://dashboard.packit.dev).
+
+As you can see in the following picture,
+the page we link on GitHub/Gitlab look now much nicer
+and the related pages are connected.
```

Comment:
This could be simplified to improve readability:

> Following picture shows a more convenient view of builds and test runs since the pages connect to each other.

Reply:
Hmmm, but it misses the visual improvements. I'll try to rephrase that.